.github/workflows/ci.yml

```diff
@@ -21,9 +21,14 @@ jobs:
       - name: Install requirements
         run: |
           python -m pip install -r requirements.txt
-          python -m pip install pytest-codestyle || true
+          python -m pip install pytest-pycodestyle || true
```

Comment:
Why keeping the `|| true` here?

Reply:
Because this still runs in python 2 and that package is not available for it.